components/core/footer/Footer.vue

```diff
@@ -5,7 +5,10 @@
                 <locale-link to="/about/code-of-conduct">
                     {{ $t('codeOfConduct') }}
                 </locale-link>
-                <!-- 
+                <locale-link to="/about/privacy-policy">
```

Comment:
The `locale-link` component is existed but commented out. Please remove the [redundant part](https://github.com/pycontw/pycontw-2021/pull/21/files#diff-0d2121e2326ba056f0ed837107dacd19d2add6dbca74f396d34808ac1c247ec5L13).